src/main/java/org/zalando/planb/revocation/api/ExceptionsResource.java

```diff
@@ -142,8 +143,7 @@ public Problem accessDenied(final AccessDeniedException e) {
     @ResponseStatus(value = HttpStatus.INTERNAL_SERVER_ERROR)
     @ResponseBody
     public Problem serializationException(final SerializationException e) {
-        log.error("A serialization exception occurred: {}", e.getMessage());
-        log.debug("Error details: ", e);
+        log.error("A serialization exception occurred", e);
```

Comment:
maybe only log like 4 or 5 stack frames? re-occuring error could really bog down the log